logger.log

```diff
@@ -0,0 +1,475 @@
+2020-10-16 14:15:38 INFO  Version:46 - HHH000412: Hibernate Core {5.4.5.Final}
```

Comment:
this file is still here
see Sofiia's tutorial on how to remove redundant files from master
and try to use commands from there considering this branch